generator/import_tracker.go

```diff
@@ -49,6 +49,7 @@ func golangTrackerLocalName(tracker namer.ImportTracker, t types.Name) string {
 	for n := len(dirs) - 1; n >= 0; n-- {
 		// follow kube convention of not having anything between directory names
 		name := strings.Join(dirs[n:], "")
+		name = strings.Replace(name, "_", "", -1)
```

Comment:
I am pedantically worried about collisions, but in practice it seems unlikely.  Any ideas on how to make this less ambiguous?

Reply:
Valid concern. But as we don't have many (quite the opposite) underscores in package names, the chances are low. And the chance of conflicts IMO is much much higher for single word package names and symbol names which do not go through a namer. E.g. let's hope nobody calls a package `in` or `out` with imported types from there.